ajax/libs/ng-toast/package.json

```diff
@@ -15,33 +14,16 @@
     "url": "git://github.com/tameraydin/ngToast.git"
   },
   "author": "Tamer Aydin (http://tamerayd.in)",
-  "contributors": [
-    {
-      "name": "Tamer Aydin",
-      "url": "http://tamerayd.in"
-    },
-    {
-      "name": "Levi Thomason",
-      "url": "http://www.levithomason.com"
-    }
-  ],
   "license": "MIT",
   "homepage": "http://tameraydin.github.io/ngToast",
-  "bugs": {
-    "url": "https://github.com/tameraydin/ngToast/issues"
-  },
-  "dependencies": {
-    "angular": ">=1.2.15 <1.6",
-    "angular-sanitize": ">=1.2.15 <1.6"
-  },
   "npmName": "ng-toast",
```

Comment:
Let's use git as our auto-update config here because there are more released versions on git.
```
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/tameraydin/ngToast.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/*"
        ]
      }
    ]
  }
```